src/hugr/validate.rs

```diff
@@ -434,21 +433,16 @@ impl<'a> ValidationContext<'a> {
         let local = Some(from_parent) == to_parent;
 
         let is_static = match from_optype.port_kind(from_offset).unwrap() {
-            // Inter-graph constant wires do not have restrictions
             EdgeKind::Static(typ) => {
-                if let OpType::Const(ops::Const(val)) = from_optype {
-                    typecheck_const(&typ, val).map_err(ValidationError::from)?;
-                } else {
-                    // If const edges aren't coming from const nodes, they're graph
-                    // edges coming from FuncDecl or FuncDefn
-                    if !OpTag::Function.is_superset(from_optype.tag()) {
-                        return Err(InterGraphEdgeError::InvalidConstSrc {
-                            from,
-                            from_offset,
-                            typ,
-                        }
-                        .into());
-                    };
+                if !(OpTag::Const.is_superset(from_optype.tag())
```

Comment:
I could add a new `OpTag::ConstSrc` here, but alternatively I could read that statement in the spec as a note that these are the only kinds of nodes which actually have a `Static` other-edgetype and remove this check and the corresponding element of `InterGraphEdgeError`....?